common/protobuf/Control.proto

```diff
@@ -0,0 +1,33 @@
+package Packet;
+
+message Control
+{
+    enum ShootMode
+    {
+        KICK = 1;
+        CHIP = 2;
+    }
+
+    enum TriggerMode
+    {
+        IMMEDIATE = 1;
+        ON_BREAK_BEAM = 2;
+        STAND_DOWN = 3;
+    }
+
+    enum Song
+    {
+        STOP = 1;
+        FIGHT_SONG = 2;
+    }
```

Comment:
Right now in the `GameplayModule`, we call `sing()` once when we score a goal.  We might need a third entry in the enum for "keep playing" so that we don't tell it to play the fight song in one packet, then tell it to stop in the next packet.